gnuboardProject/frontend/src/components/Portfolio/PortfolioForm.tsx

```
@@ -99,6 +188,23 @@ const PortfolioForm: React.FC = () => {
   return (
     <form onSubmit={handleSubmit} style={{maxWidth:700,margin:'2rem auto',background:'#fff',borderRadius:12,boxShadow:'0 2px 8px rgba(0,0,0,0.07)',padding:'2rem'}}>
       <h2 style={{fontSize:'1.5rem',fontWeight:700,marginBottom:'1.5rem'}}>{id ? '포트폴리오 수정' : '포트폴리오 작성'}</h2>
+      <div style={{marginBottom:'2rem'}}>
+        <label style={{display:'block',marginBottom:'1rem',fontWeight:600}}>템플릿 선택</label>
+        <div style={{display:'grid',gridTemplateColumns:'repeat(3,1fr)',gap:'1rem'}}>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)